packages/writing-style/.gitignore

```diff
@@ -1,2 +1,3 @@
 *.zip
 *.tar.gz
+vale-*
```

Comment:
I'm not sure we need this. You just need to remove your binary file from disk, as it's moved to another location.

Reply:
@nkuehn and I discussed it...

Reply:
Hmm ok